SplunkRumWorkspace/SplunkRum/SplunkRum/Session.swift

```diff
@@ -18,9 +18,11 @@ limitations under the License.
 import Foundation
 
 let MAX_SESSION_AGE_SECONDS = 4 * 60 * 60
+let MAXSESSIONINACTIVEAGESECONDS = 15 * 60
```

Comment:
Snake-case this and name it appropriately please.  `INACTIVITY_SESSION_TIMEOUT_SECONDS`